winui/Shimmer/Built-in-Types.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Built-in Types of WinUI Shimmer Control | Syncfusion
+description: Learn about the various built-in types available in the Syncfusion WinUI Shimmer(SfShimmer) control, including Circle Persona, Square Persona, Profile, Article, Video, Feed, and Shopping.
+platform: WinUI
```

Comment:
@GilbertmanojSF3971 

title: Built-in types of WinUI Shimmer control | Syncfusion
description: Learn about the various built-in types available in the Syncfusion WinUI Shimmer (SfShimmer) control, including CirclePersona, SquarePersona, Profile, Article, Video, Feed, and Shopping.

Reply:
@Manivannan-E Sir. Changed the title and description content.